pipeline/scripts/download-electron-mirror.js

```diff
@@ -1,26 +1,51 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 const pkg = require('../../package.json');
-const download = require('electron-download');
```

Comment:
Can we drop the electron-download dev dependency?

Reply:
Yes, I think so, good catch. electron-chromedriver uses it but we should be able to remove it from our explicit devDependency list. Will try